packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -95,20 +96,28 @@ export class FocusTrapZone extends BaseComponent<IFocusTrapZoneProps, {}> implem
         ref={ this._root }
         aria-labelledby={ ariaLabelledBy }
         onKeyDown={ this._onKeyboardHandler }
+        onFocusCapture={ this._onFocusCapture }
       >
         { this.props.children }
       </div>
     );
   }
 
-  /**
-   * Need to expose this method in case of popups since focus needs to be set when popup is opened
-   */
   public focus() {
-    const { firstFocusableSelector } = this.props;
-    const focusSelector = typeof firstFocusableSelector === 'string'
-      ? firstFocusableSelector
-      : firstFocusableSelector && firstFocusableSelector();
+    const { focusTransferRule = FocusTransferRule.firstFocusable, firstFocusableSelector } = this.props;
```

Comment:
You should probably use defaultProps here in case someone else creates another method and needs to use focusTransferRule

Reply:
done.